Ginger/Ginger/GeneralValidationRules/MaxLengthValidationRule.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Controls;
+
+namespace Amdocs.Ginger.ValidationRules
+{
+    public class MaxLengthValidationRule : ValidationRule
+    {       
+        string mMessage;
+        string Message
+        {
+            get
+            {
+                if (string.IsNullOrEmpty(mMessage))
+                {
+                    return "Maximum Length allowed is {0}";
```

Comment:
How this {0} will work? was it tested to provide a good message? if yes add comment in code

Reply:
this is formatted message, when returning validationresult i'm using string format which replace {0} with some Max Length